fuse/data/ops/ops_aug_tabular.py

```diff
@@ -0,0 +1,103 @@
+import random
+import numpy as np
+from typing import Optional, Any, Sequence
+
+from fuse.data.ops.op_base import OpBase
+
+from fuse.utils import NDict
+
+
+class OpAugOneHot(OpBase):
+    """
```

Comment:
Cool, just update the documentation - cause now you don't get prob.
Instead, show examples with OpRandApply.